t/fixtures/with_same_distribution/cpanfile.snapshot

```diff
@@ -0,0 +1,191 @@
+# carton snapshot format: version 1.0
```

Comment:
I'm not sure how to generate `cpanfile.snapshot` file with `Test-Class-0.49` and `Test-Simple-1.302175`. I copied this file from `t/fixtures/simple/cpanfile` and edited manually.

Reply:
You can run `cd t/fixtures/with_same_distribution/ && carton install && rm -rf local/` to generate `cpanfile.snapshot`.
Manual editing is also fine since CI is passing.